Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -26,10 +26,14 @@ extension KeyboardShortcuts {
 	```
 	*/
 	public final class RecorderCocoa: NSSearchField, NSSearchFieldDelegate {
-		private let minimumWidth: Double = 130
 		private var eventMonitor: LocalEventMonitor?
 		private let onChange: ((_ shortcut: Shortcut?) -> Void)?
 		private var observer: NSObjectProtocol?
+		private var intrinsicWidth: CGFloat = 130
+		
+		private let intrinsicWidthRegular: CGFloat = 130
+		private let intrinsicWidthSmall: CGFloat = 110
+		private let intrinsicWidthMini: CGFloat = 95
```

Comment:
We should not hard-code the minimum widths, since we support localization. They should be based on the intrinsic content size of the labels in the button, with some tiny padding on each side.